releasenotes/notes/ambient-dns-on.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+releaseNotes:
+  - |
+    **Promoted** the `cni.ambient.dnsCapture` value to default to `true`.
+    This enables the DNS proxying for workloads in ambient mesh by default, improving security, performance, and enabling
+    a number of features.
```

Comment:
We should add an `upgradeNote` here to explicitly indicate that a pod restart is required after upgrade for this to take effect (unless we intend to also merge https://github.com/istio/istio/pull/53906 in 1.25, which will make that moot)